app/views/wikis/edit.html.erb

```diff
@@ -10,11 +10,7 @@
       <%= f.label :body %>
       <%= f.text_area :body, rows: 10, class: 'form-control', placeholder: "Enter Wiki Body" %>
     </div>
-    <div class="form-group">
-      <%= f.label :private, class: 'checkbox' do %>
-        <%= f.check_box :private %> Private Wiki
-      <% end %>
-    </div>
+      <%= render partial: "wikis/form" %>
```

Comment:
Here is where the undefined method on `f` is happening. It's also happening within the create view (same setup). @bjblock